src/_repobee/ext/gitlab.py

```diff
@@ -187,15 +185,11 @@ def _ensure_members_in_team(
 
         if missing_members:
             LOGGER.info(
-                "Adding members {} to team {}".format(
-                    ", ".join(missing_members), team.name
-                )
+                f'Adding members {", ".join(missing_members)} to team {team.name}'
             )
         if existing_members:
             LOGGER.info(
-                "{} already in team {}, skipping...".format(
-                    ", ".join(existing_members), team.name
-                )
+                f'{", ".join(existing_members)} already in team {team.name}, skipping...'
```

Comment:
Also just slightly too long.
```suggestion
                f'{", ".join(existing_members)} already in team {team.name}, '
                f"skipping..."
```